nx-X11/config/cf/linux.cf

```diff
@@ -733,6 +733,21 @@ XCOMM binutils:	(LinuxBinUtilsMajorVersion)
 # define ServerExtraDefines	-DGCCUSESGAS XFree86ServerDefines
 #endif
 
+#ifdef MipselArchitecture
+# ifndef OptimizedCDebugFlags
+#  define OptimizedCDebugFlags	DefaultGcc2MipsOpt
+# endif
+# define LinuxMachineDefines	-D__MIPSEL__
+# define ServerOSDefines	XFree86ServerOSDefines -DDDXTIME
+# ifdef Mips64elArchitecture
```

Comment:
You could have probably merged this into the `Mips` block and checked for `Mips(64)elArchitecture` when needed within it. Would have made some stuff more compact, but maybe also more difficult to read. Your call.